examples/simple_zonal_with_asm/hub.tf

```diff
@@ -14,11 +14,21 @@
  * limitations under the License.
  */
 
-module "hub" {
-  source                  = "../../modules/hub"
-  project_id              = var.project_id
-  location                = module.gke.location
-  cluster_name            = module.gke.name
-  cluster_endpoint        = module.gke.endpoint
-  gke_hub_membership_name = "gke-asm-membership"
+resource "google_gke_hub_feature" "mesh_feature" {
```

Comment:
This should probably be part of the module.

Reply:
Agreed, blocked on the provider bug described below.